001-devcycle.conf.template

```diff
@@ -1,25 +1,11 @@
-WSGIPythonPath usr/local/devcycle
```

Comment:
Yeah, I think it's worthwhile. It's super easy and I got it working in a couple minutes so I'll get that pushed up.